containers/wordpress/wp_opt/https/files/nginx.conf.in

```diff
@@ -0,0 +1,125 @@
+worker_processes  96; # Depends on #vcpus
+
+error_log __NGINX_TEMP_DIR__/nginx-error.log;
+pid __NGINX_PID_FILE__;
+
+worker_rlimit_nofile 1000000;
+
```

Comment:
missing "ssl_engine" block from the wp_opt in nginx.conf.in file.

Reply:
done